src/Print.jsx

```diff
@@ -36,24 +36,36 @@ export default class Print extends React.Component {
     if (this.props.main || this.props.single) {
 
       window.matchMedia('print').onchange = () => {
-        const bodyRect = document.body.getBoundingClientRect();
-        const elem = this.printElement;
-        const elemRect = elem && elem.getBoundingClientRect();
-        const printOffsetLeft = elemRect && (elemRect.left - bodyRect.left);
-        const printOffsetTop = elemRect && (elemRect.top - bodyRect.top);
+        const isPrint = window.matchMedia('print').matches;
 
-        this.setState({
-          printOffsetTop,
-          printOffsetLeft,
-        });
+        if(isPrint){
+          console.log('PRINT HERE');
+          const bodyRect = document.body.getBoundingClientRect();
+          const elem = this.printElement;
+          const elemRect = elem && elem.getBoundingClientRect();
+          const printOffsetLeft = elemRect && (elemRect.left - bodyRect.left);
+          const printOffsetTop = elemRect && (elemRect.top - bodyRect.top);
+
+          this.setState({
+            printOffsetTop,
+            printOffsetLeft,
+          });
+        }else{
+          console.log('print end');
+          this.setState({
+            printOffsetTop: 0,
+            printOffsetLeft: 0
```

Comment:
? R u remember about:
> unexpected initial margin-top: 0px;margin-left: 0px;
as described at errata:
https://github.com/a-x-/react-easy-print/pull/19#issuecomment-398208526

you should avoid 0 0 margins

Reply:
we do avoid them ` const offset_ = ((printOffsetTop || printOffsetLeft ) && main_ && isPrint) ? { marginTop: -printOffsetTop, marginLeft: -printOffsetLeft} : {};
`

Reply:
o-o-ok :-)